Artman cannot execute pipelines.

I just pulled `master` from toolkit, and now Artman cannot successfully build a GAPIC anymore:

```
(pipeline) lukesneeringer@lukesneeringer0:~/Code/Google/artman:master$ ./execute_pipeline.py --config "../googleapis/gapic/api/artman_speech.yaml,../googleapis/gapic/lang/common.yaml" --language python GapicClientPipeline 
Final args:
  gapic_language_yaml : ['../googleapis/gapic/lang/python_gapic.yaml']
  proto_deps : ['google-common-protos']
  repo_root : ..
  package_dependencies_yaml : ../googleapis/gapic/packaging/dependencies.yaml
  language : python
  stage_output : False
  import_proto_path : ['../googleapis']
  common_protos_yaml : ../googleapis/gapic/packaging/common_protos.yaml
  organization_name : google-cloud
  final_repo_dir : ../artman/output/gcloud-python-cloud-speech
  api_name : speech
  output_dir : ../artman/output
  toolkit_path : ../toolkit
  package_defaults_yaml : ../googleapis/gapic/packaging/api_defaults.yaml
  staging_repo_dir : ../api-client-staging/generated
  src_proto_path : ['../googleapis/google/cloud/speech/v1']
  service_yaml : ['../googleapis/google/cloud/speech/cloud_speech.yaml']
  api_version : v1
  gapic_api_yaml : ['../googleapis/google/cloud/speech/v1/cloud_speech_gapic.yaml']
Create GapicClientPipeline instance.
Compiling descriptors for ['../googleapis/google/cloud/speech/v1/cloud_speech.proto']
Searching for latest protobuf source
google/cloud/speech/v1/cloud_speech.proto: warning: Import google/protobuf/any.proto but not used.
google/cloud/speech/v1/cloud_speech.proto: warning: Import google/protobuf/duration.proto but not used.

:buildSrc:compileJava UP-TO-DATE
:buildSrc:compileGroovy UP-TO-DATE
:buildSrc:processResources UP-TO-DATE
:buildSrc:classes UP-TO-DATE
:buildSrc:jar UP-TO-DATE
:buildSrc:assemble UP-TO-DATE
:buildSrc:compileTestJava UP-TO-DATE
:buildSrc:compileTestGroovy UP-TO-DATE
:buildSrc:processTestResources UP-TO-DATE
:buildSrc:testClasses UP-TO-DATE
:buildSrc:test UP-TO-DATE
:buildSrc:check UP-TO-DATE
:buildSrc:build UP-TO-DATE

FAILURE: Build failed with an exception.

* Where:
Build file '/usr/local/google/home/lukesneeringer/Code/Google/toolkit/build.gradle' line: 48

* What went wrong:
A problem occurred evaluating root project 'toolkit'.
> version.txt (No such file or directory)

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

The problem seems to be this line added to `build.gradle` in 758ecf343:

    project.version = new File("version.txt").text.trim()

The problem is that my working directory is not the root directory of toolkit (it is the root of Artman). This probably needs to be explicitly made relative to the location of the `build.gradle` file.

--- a/build.gradle
+++ b/build.gradle
@@ -45,7 +45,7 @@ apply plugin: "signing"
 sourceCompatibility = 1.7
 
 group = "com.google.api"
-project.version = new File("version.txt").text.trim()
+project.version = file("version.txt").text.trim()
 
 def protoVersion = "3.0.0-beta-3"
 

